harvestingkit/bibrecord.py

```diff
@@ -548,6 +548,7 @@ def record_add_field(rec, tag, ind1=' ', ind2=' ', controlfield_value='',
                 immediate_lower_tag = '000'
                 for rec_tag in rec:
                     if (tag not in ('FMT', 'FFT', 'BDR', 'BDM') and
+                            rec[rec_tag] and
```

Comment:
I don't have enough (time) knowledge about the harvesting kit to dig deeper in the issue, I think it might be also related with the fact that we remove some fields from the records because we don't what them in the output xml.